src/auth/reset-token.service.ts

```diff
@@ -0,0 +1,33 @@
+import { Injectable } from "@nestjs/common";
+import { InjectRepository } from "@nestjs/typeorm";
+import { ResetTokenEntity as ResetToken, ResetTokenEntity} from "./reset-token.entity";
+import { MoreThan, Repository } from "typeorm";
+import { ConfigService } from "@nestjs/config";
+
+@Injectable()
+export class ResetTokenService{
+    constructor(
+        @InjectRepository(ResetToken) private readonly resetTokenRepository: Repository<ResetToken>,
+        private readonly configService: ConfigService
+    ){}
+
+    async create(user_id: number, token: string): Promise<void>{
+        const expiredAfterMinutes = Number(this.configService.get('RESET_TOKEN_EXPIRY'));
```

Comment:
const expiredAfterMinutes = Number(this.configService.get('RESET_TOKEN_EXPIRY'));
        const expires_in = new Date();
        expires_in.setMinutes(expires_in.getMinutes() + expiredAfterMinutes); 

        Create Separate method